v2/AdminCore.WebApp/Models/Holiday/CreateHolidayViewModel.cs

```diff
@@ -1,10 +1,14 @@
-﻿using AdminCore.Constants.Enums;
+﻿using System.Collections;
+using System.Collections.Generic;
+using AdminCore.Constants.Enums;
 using AdminCore.DAL.Models;
 
 namespace AdminCore.WebApi.Models.Holiday
 {
   public class CreateHolidayViewModel
   {
-    public EventTypes EventType { get; set; }
+    public DateViewModel DateRange { get; set; }
```

Comment:
What is this used for? If you are creating a holiday the controller would accept start and end date and then you'd pass the employee Id using the authenticated user service.